src/components/ImportForm/ComponentSection/ComponentSection.tsx

```diff
@@ -28,6 +76,16 @@ export const ComponentSection = () => {
         isRequired
         data-testid="component-name"
       />
+      <ExpandableSection
+        toggleTextExpanded="Hide advanced annotation options"
+        toggleTextCollapsed="Show advanced annotation options"
+        isExpanded={isAnnotationExpanded}
+        data-test="expand-annotation-section"
+        onToggle={() => setAnnotationExpanded((x) => !x)}
+      >
+        <GitProviderDropdown name="gitProviderAnnotation" />
+        <InputField name="gitURLAnnotation" label="Git url annotation" data-test="url-annotation" />
+      </ExpandableSection>
```

Comment:
We don't need a separate expandable section, you can move these two fields under the Advanced git options

Reply:
updated